NitroxClient/GameLogic/InitialSync/PlayerInitialSyncProcessor.cs

```diff
@@ -66,17 +66,14 @@ private void SetPlayerStats(PlayerStatsData statsData)
         {
             if (statsData != null)
             {
-                using (packetSender.Suppress<PlayerStats>())
+                Player.main.oxygenMgr.AddOxygen(statsData.Oxygen);
+                Player.main.liveMixin.health = statsData.Health;
+                Player.main.GetComponent<Survival>().food = statsData.Food;
+                Player.main.GetComponent<Survival>().water = statsData.Water;
+                Player.main.infectedMixin.SetInfectedAmount(statsData.InfectionAmount);
+                if (statsData.InfectionAmount > 0f)
                 {
-                    Player.main.oxygenMgr.AddOxygen(statsData.Oxygen);
-                    Player.main.liveMixin.health = statsData.Health;
-                    Player.main.GetComponent<Survival>().food = statsData.Food;
-                    Player.main.GetComponent<Survival>().water = statsData.Water;
-                    Player.main.infectedMixin.SetInfectedAmount(statsData.InfectionAmount);
-                    if (statsData.InfectionAmount > 0f)
-                    {
-                        Player.main.infectionRevealed = true;
-                    }
+                    Player.main.infectionRevealed = true;
```

Comment:
If you look into Subnautica code this can probably lead to the "Infection-Reveal-Sequence" not being played now that we changed the initial value.
![grafik](https://user-images.githubusercontent.com/23176718/99871075-0151cb00-2bd8-11eb-9786-88123a330299.png)